src/screens/TopicPageScreen/index.js

```diff
@@ -368,7 +381,7 @@ const TopicPageScreen = (props) => {
         setIsHeaderHide(true);
       }
     },
-    [animatedHeight]
+    [animatedHeight, coverPath]
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The dependency array for the `handleOnScroll` function now includes `coverPath`. This ensures that the function is updated whenever `coverPath` changes, keeping the behavior consistent with the current state of `coverPath`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->